_posts/2020-06-25-huechong-hettrick.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Software is the backbone"
+author: "@pweschmidt"
+rse: "Neil Chue Hong and Simon Hettrick"
+phenotype: neilsimon-rse-phenotype.png
+excerpt: "For ca 10 years the Software Sustainability Institute in the UK has been the backbone of research enginnering. But how did they fare during the coronavirus pandemic and how will research software engineering change in future?"
+date: 2020-06-11 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-huechong-hettrick-episode-22.mp3
+length: 18840264
+duration: "00:39:16"
+explicit: "no"
+resources:
+ - name: The Software Sustainability Institute
+   url:  https://www.software.ac.uk
+ - name: History of Research Software Engineering in the UK
+   url: https://www.software.ac.uk/blog/2016-08-17-not-so-brief-history-research-software-engineers
+ - name: Software Sustainability Institute on Twitter
+   url: https://twitter.com/SoftwareSaved 
+ - name: Neil Chue Hong
+   url: https://uk.linkedin.com/in/neilchuehong
+ - name: Simon Hettrick
+   url: https://uk.linkedin.com/in/simon-hettrick-2668a9a
+--- 
+
+In today's episode Peter meets Neil Chue Hong and Simon Hettrick. Neil and Simon are the director/deputy director of the Software Sustainability Institute (https://www.software.ac.uk). For ca 10 years the institute has been the "backbone" for research software engineering and the RSE movement. 
+It helped build the RSE movement and network, not only in the UK but in other countries. Neil and Simon discuss what changes the coronavirus pandemic brought to research software engineering. And what challenges and opportunities lie ahead of us. And did you know the Institut has a blog with cooking recipes?
```

Comment:
Probably you could do an and, and then continue the sentence )combine these two).

Reply:
Done